### ISSUE TITLE
Handling of HTML inside BibTex from Crossref

### ISSUE THREAD

**Issue by LocoDelAssembly** [SpeciesFileGroup/taxonworks#1142]
Normally, BibTex format is pure plaintext, but there are some cases like https://search.crossref.org/?q=10.11646%2Fzootaxa.4674.4.8 which produce the following output:

BibTex
=====
```BibTex
@article{FENG_2019,
	doi = {10.11646/zootaxa.4674.4.8},
	url = {https://doi.org/10.11646%2Fzootaxa.4674.4.8},
	year = 2019,
	month = {sep},
	publisher = {Magnolia Press},
	volume = {4674},
	number = {4},
	pages = {491--495},
	author = {XUELI FENG and SHIHUI HUANG and CHANGQING LUO},
	title = {$\less$p$\greater$$\less$strong$\greater$A new species of the subgenus $\less$em$\greater$Tachycines$\less$/em$\greater$ ($\less$em$\greater$Gymnaeta$\less$/em$\greater$) (Orthoptera:~$\less$/strong$\greater$$\less$strong$\greater$Rhaphidophoridae) from karst caves of southern Guizhou, China$\less$/strong$\greater$$\less$/p$\greater$},
	journal = {Zootaxa}
}
```
Chicago
======
```html
FENG, XUELI, SHIHUI HUANG, and CHANGQING LUO. “<p><strong>A New Species of the Subgenus <em>Tachycines</em> (<em>Gymnaeta</em>) (Orthoptera: </strong><strong>Rhaphidophoridae) from Karst Caves of Southern Guizhou, China</strong></p>.” Zootaxa 4674, no. 4 (September 26, 2019): 491–495. doi:10.11646/zootaxa.4674.4.8.
```
As can be seen, those `$\less$`/`$\greater$` are encoding HTML tags.

Things to do:

[] Research whether this is actually valid BibTex.
[] If above is true write code to convert to HTML, then convert to plaintext as a second step (or should we support HTML?).

Currently, this is the result of submitting 10.11646/zootaxa.4674.4.8 to create source from citation:

<img width="845" alt="image" src="https://user-images.githubusercontent.com/4991925/67116781-90abd400-f1b7-11e9-9c62-3b42e08ddc1a.png">

Code seems to be stripping out `$`already, but leaving commands as-is.

**Comment by proceps** [comment]
I tend to manually edit those entries at the moment.

**Comment by mjy** [comment]
We can strip all but <i> tags. Definitely don't want html there

M

**Comment by typophyllum** [comment]
The problem is still occurring with Zootaxa papers:

![imagen](https://user-images.githubusercontent.com/33210201/75458580-ce3a1e00-595c-11ea-9134-25be5366a45b.png)

**Comment by mjy** [comment]
That's actually not HTML, but Bibtex formatting. That's going to be very hard to parse, almost like we need to talk to Crossref about the issue.

**Comment by LocoDelAssembly** [comment]
It kinda is, _\less_**p**_\greater_ would be _<_**p**_>_ And I suspect we actually receive $\less$ but it ends up unrecognized and is rendered as literal.

Crossref seems to support https://support.crossref.org/hc/en-us/articles/213126186-NLM-JATS-abstracts but there is no indication that may be used elsewhere. And it is still totally unclear if this is supposed to be legal BibTex or they are making it up. At the very least using paragraph tag for something that is supposed to be a title is odd. And < strong > probably should be \textbf or similar.

**Comment by mjy** [comment]
I suppose we can add some front-end filters to strip it. 

Remember folks- Crossref data come from users, who can do crazy stuff, it's NOT unified.

**Comment by typophyllum** [comment]
In this case I copied the original Zootaxa DOI. With other journals it works fine.

**Comment by LocoDelAssembly** [comment]
Pushed change. Might consider catch non-oddly encoded HTML tags and remove them if they're not italics.
